test/tracerEq/test_h-advection_mes_2d.py

```diff
@@ -47,6 +47,9 @@ def run(refinement, **model_options):
     options.output_directory = outputdir
     options.simulation_end_time = t_end
     options.simulation_export_time = t_export
+    solverobj.create_function_spaces()
```

Comment:
I wonder whether it's worth adding an example on this functionality - but not essential for now 👍